python/python/tests/test_dataset.py

```diff
@@ -4778,17 +4805,17 @@ def test_shallow_clone(tmp_path: Path):
     ds = lance.write_dataset(table_v2, src_dir, mode="overwrite")
 
     # Create a tag pointing to version 1
-    ds.tags.create("v1", 1, None)
+    ds.tags.create("v1", (None, 1))
```

Comment:
Add a case to verify if we are not in the `main` branch, but create a tag point to the main branch?